lib/puppet/http/service/ca.rb

```diff
@@ -0,0 +1,49 @@
+class Puppet::HTTP::Service::Ca < Puppet::HTTP::Service
+  HEADERS = { 'Accept' => 'text/plain' }.freeze
+
+  def get_certificate(name, ssl_context: nil)
```

Comment:
Seems like Puppet::Rest::routes.get_certificates had an extra logging message `Puppet.info _("Downloaded certificate for %{name} from %{server}") % { name: name, server: ca.server }` - https://github.com/puppetlabs/puppet/blob/master/lib/puppet/rest/routes.rb#L43
This is causing the failures in `acceptance/tests/ssl/autosign_command.rb`
```    lax-vice.delivery.puppetlabs.net (lax-vice.delivery.puppetlabs.net) executed in 7.01 seconds
Beaker::DSL::Outcomes::FailTest:
Expected certificate to be autosigned.
Expected /Downloaded certificate for unfunny-vinegar.delivery.puppetlabs.net/ to match "\e[0;32mInfo: Resolving service 'ca' using Puppet::HTTP::Resolver::Settings\e[0m\n\e[0;32mInfo: Resolved service 'ca' to https://lax-vice.delivery.puppetlabs.net:8140/puppet-ca/v1\e[0m\n\e[0;32mInfo: Creating a new RSA SSL key for unfunny-vinegar.delivery.puppetlabs.net-autosign\e[0m\n\e[0;32mInfo: csr_attributes file loading from /etc/puppetlabs/puppet/csr_attributes.yaml\e[0m\n\e[0;32mInfo: Creating a new SSL certificate request for unfunny-vinegar.delivery.puppetlabs.net-autosign\e[0m\n\e[0;32mInfo: Certificate Request fingerprint (SHA256): 59:CD:6E:2D:15:0B:3F:51:3E:9A:23:26:76:33:70:63:36:1E:9E:ED:C2:16:02:85:A5:B7:93:3B:6A:77:64:75\e[0m\n\e[0;32mInfo: Using configured environment 'production'\e[0m\n\e[0;32mInfo: Retrieving pluginfacts\e[0m\n\e[0;32mInfo: Retrieving plugin\e[0m\n\e[0;32mInfo: Retrieving locales\e[0m\n\e[0;32mInfo: Caching catalog for unfunny-vinegar.delivery.puppetlabs.net-autosign\e[0m\n\e[0;32mInfo: Applying configuration version '1573566972'\e[0m\n\e[mNotice: Applied catalog in 0.01 seconds\e[0m\n".```